src/cmd/flux-mini.py

```diff
@@ -176,6 +177,12 @@ def create_parser(self):
             action="store_true",
             help="Don't actually submit job, just emit jobspec",
         )
+        parser.add_argument(
+            "-d",
+            "--debug-emulate",
```

Comment:
Consider making this a long-only option, and hide it from default output with `help=argparse.SUPPRESS`

Reply:
Sure.